app/src/main/java/com/kickstarter/ui/adapters/ActivityFeedAdapter.java

```diff
@@ -29,12 +30,12 @@
   private static final int SECTION_SURVEYS_VIEW = 3;
   private static final int SECTION_ACTIVITIES_VIEW = 4;
 
-  private final Delegate delegate;
+  private final @Nullable Delegate delegate;
 
   public interface Delegate extends FriendBackingViewHolder.Delegate, ProjectStateChangedPositiveViewHolder.Delegate,
     ProjectStateChangedViewHolder.Delegate, ProjectUpdateViewHolder.Delegate, EmptyActivityFeedViewHolder.Delegate, UnansweredSurveyViewHolder.Delegate {}
 
-  public ActivityFeedAdapter(final @NonNull Delegate delegate) {
+  public ActivityFeedAdapter(final @Nullable Delegate delegate) {
```

Comment:
I just pushed this change to the branch.

basically, I think our delegates should always be optional (much like we do in iOS) and that way we don't have to use them in tests and put in an empty implementation.